### DISCUSSION TITLE
How to get a simliar functionality as loc in pandas

### DISCUSSION THREAD

**Discussion by derrickcchow** [vaexio/vaex#1867] [Q&A] [answered]
Hi,
I was wondering if there is a way to implement something similar to loc in pandas in vaex.
This is the code:  df.loc[df["Variable Value"] == x, columnname] = value

thanks, Derrick

**Comment by yohplala** [comment, marked as answer]
> Hi, I was wondering if there is a way to implement something similar to loc in pandas in vaex. This is the code: df.loc[df["Variable Value"] == x, columnname] = value
> 
> thanks, Derrick

Hi, for such a pattern, you should head to `where`.
From API documentation (I would have used a link, would there be one for individual function)

![Screenshot from 2022-01-27 21-41-51](https://user-images.githubusercontent.com/13206058/151440378-34e3153f-ce0f-41fa-a533-f769e2ffaf8d.png)

Bests
